pkg/teleporter/relayer.go

```diff
@@ -24,8 +24,9 @@ import (
 	"github.com/ava-labs/avalanche-cli/pkg/models"
 	"github.com/ava-labs/avalanche-cli/pkg/utils"
 	"github.com/ava-labs/avalanche-cli/pkg/ux"
-	"github.com/ava-labs/awm-relayer/config"
+	apiConfig "github.com/ava-labs/awm-relayer/config"
```

Comment:
why not call it relayerconfig

Reply:
This really is just apiConfig. The context here is that the awm-relayer defines two applications now `awm-relayer` and `signature-aggregator` these share some config values which are defined in the base config above so aliasing it to `apiConfig` or `baseConfig` is appropriate.

Notice that `"github.com/ava-labs/awm-relayer/relayer/config"` is included below separately

Reply:
I  think I can resolve this one